test/performance/Makefile.am

```diff
@@ -25,7 +25,10 @@ COMPILE_ONLY = odp_dma_perf \
 	       odp_scheduling \
 	       odp_timer_perf
 
-TESTSCRIPTS = odp_dma_perf_run.sh \
+TESTSCRIPTS = odp_cpu_bench_run.sh \
+	      odp_crypto_run.sh \
+              odp_dma_perf_run.sh \
```

Comment:
Seems to be all whitespaces in this line.

Reply:
will fix